Sources/Basics/Observability.swift

```diff
@@ -597,6 +608,8 @@ extension TSCBasic.Diagnostic {
             self = .init(message: .error(data), location: location)
         case .warning:
             self = .init(message: .warning(data), location: location)
+        case .output:
+            self = .init(message: .note(data), location: location)
```

Comment:
Currently mapped to `note`, just as `info` is — changing this will require TSCBasic to add this case first.